@xen-orchestra/web-core/lib/components/ui/dropdown-button/UiDropdownButton.vue

```diff
@@ -0,0 +1,81 @@
+<!-- v3 -->
+<template>
+  <button type="button" class="ui-dropdown-item" :class="{ selected }" :disabled="isDisabled">
+    <VtsIcon :icon="icon" accent="current" class="left-icon" fixed-width />
```

Comment:
Prefer the shorthand version:
```suggestion
    <VtsIcon :icon accent="current" class="left-icon" fixed-width />
```